src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Driver.java

```diff
@@ -0,0 +1,159 @@
+package cz.muni.pa165.teamwhite.formula1.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+
+/**
+ * @author Tomas Sedlacek
+ */
+
+@Entity
+public class Driver {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Car car;
+
+    @Column(nullable = false)
+    @NotNull
+    private String name;
+
+    @Column(nullable = false)
+    @NotNull
+    private String surname;
+
+    @Column(nullable = false)
+    @NotNull
+    private String nationality;
+
+    @Column(nullable = false)
+    @NotNull
+    private boolean isAgresive;
+
+    @Column(nullable = false)
+    @NotNull
+    private int wetDriving;
+
+    @Column(nullable = false)
+    @NotNull
+    private int reactions;
+
+
+    public Driver() {}
+
+    public Driver(Long id) {
+        this.id = id;
+    }
+
+    public Driver(Long id, Car car, String name, String surname, String nationality,
```

Comment:
Chceme tady to ID? my jsme id do konstruktoru nedavali... myslenka za tim byla takova, ze toto je konstruktor, ktery chces v aplikaci idealne pouzivat kdyz znas vsechny hodnoty. To bys ale nemohl, kdyz tento konstruktor vyzaduje ID.